DFC.ServiceTaxonomy.ApiFunction/CypherQueries/GetJobProfileByTitle.json

```diff
@@ -0,0 +1,8 @@
+﻿{
+  "query": "MATCH (dtd:ncs__DayToDayTask)<-[:ncs__hasDayToDayTask]-(jp:ncs__JobProfile{skos__prefLabel:$canonicalName})-[:ncs__hasSocCode]->(soc:ncs__SOCCode), (jp)-[:ncs__relatedOccupation]-(oc:esco__Occupation) OPTIONAL MATCH (jp)-[:ncs__hasWorkingUniform]-(wu:ncs__WorkingUniform) OPTIONAL MATCH (jp)-[:ncs__hasWorkingEnvironment]-(we:ncs__WorkingEnvironment) OPTIONAL MATCH (jp)-[:ncs__hasWorkingLocation]-(wl:ncs__WorkingLocation) WITH wu,we,wl,jp,soc,oc, {title: jp.skos__prefLabel, dayToDayTasks:collect(dtd.ncs__Description)} as combinedProfiles RETURN {Title:combinedProfiles.title, LastUpdatedDate:'ToDo', Url:jp.ncs__JobProfileWebsiteUrl, Soc:soc.skos__prefLabel, ONetOccupationalCode:'ToDo', AlternativeTitle:REDUCE(s = HEAD(oc.skos__altLabel), n IN TAIL( oc.skos__altLabel) | s + ', ' + n), Overview:jp.ncs__Description, SalaryStarter:jp.ncs__SalaryStarter, SalaryExperienced:jp.ncs__SalaryExperienced, MinimumHours:jp.ncs__MinimumHours, MaximumHours:jp.ncs__MaximumHours, WorkingHoursDetails:jp.ncs__WorkingHoursDetails, WorkingPattern:jp.ncs__WorkingPattern, WorkingPatternDetails:jp.ncs__WorkingPatternDetails, HowToBecome:{EntryRouteSummary:'ToDo', EntryRoutes:{University:{RelevantSubjects:['ToDo'], FurtherInformation:['ToDo'], EntryRequirementPreface:['ToDo'], EntryRequirements:['ToDo'], AdditionalInformation:['ToDo']}}, MoreInformation:{Registration:['ToDo'], CareerTips:[jp.ncs__HtbCareerTips], ProfessionalAndIndustryBodies:[jp.ncs__HtbBodies], FurtherInformation:[jp.ncs__HtbFurtherInformation]}}, WhatItTakes:{DigitalSkillsLevel:jp.ncs__WitDigitalSkillsLevel, Skills:['ToDo']}, WhatYouWillDo:{ WYDDayToDayTasks:combinedProfiles.dayToDayTasks, WorkingEnvironment:{Location:COALESCE(wl.ncs__Description, ''), Environment:COALESCE(we.ncs__Description, ''), Uniform:COALESCE(wu.ncs__Description, '') }}, CareerPathAndProgression:{CareerPathAndProgression:[jp.ncs__CareerPathAndProgression]}, RelatedCareers:['ToDo']}",
```

Comment:
I think the relationship to ncs__DayToDayTask should be optional.
Some JobProfiles don't seem to have any tasks.